### DISCUSSION TITLE
Where can I change the file to add my own image?

### DISCUSSION THREAD

**Discussion by FrankTheTank908** [Berny23/LD-ToyPad-Emulator#143] [Q&A]
Hello everyone,

where can I change the file that would be able to show my personal image, instead of text?

I find it very hard to play with my kids, where with the Lego pieces I have personally, I can pick it out and play right away. 

This would be my own images and not something copyrighted at all.

thanks,
Frank

**Comment by FrankTheTank908** [comment]
For example I take a picture with my camera, add it to a image hoster and point to the image to be automatically cropped to a usable image size and use in webUI

**Reply by dverzolla** [reply]
Everything is inside the 'server' folder (https://github.com/Berny23/LD-ToyPad-Emulator/tree/master/server).
You must be familiar with basic web development to achieve what you want.

**Comment by Lypsilonx** [comment]
I would also want to do this... Maybe I'll go have a look around the code when my PI arrives and I can test...

**Reply by Lypsilonx** [reply]
I've made a pull request now: https://github.com/Berny23/LD-ToyPad-Emulator/pull/208

**Reply by FrankTheTank908** [reply]
Thank you for contributing this! Last year I lost hope on maybe adding images. And I stoped playing the game because my son lost interest but now I'm starting to feel like I should go back and finish this huge game. As I was having lots of fun.

Plus my modded Wii U is just sitting there, and I need to remake the pi image as it was repurposed for another project. But I have another one now.

Now it's going to take me forever to find all the images and convert them 🤣 (in need of some energy drinks and some quiet time away from the kids for awhile to do this image finding and convert images project) mailto:Frankymouse1991@gmail.com if someone has any suggestions if not allowed on GitHub
